types/membership.js

```diff
@@ -0,0 +1,35 @@
+const { GroupData } = require('./groupData')
+const Serializable = require('./serializable')
+
+class Membership extends Serializable {
```

Comment:
Prefer GroupMembership for this class so its clear its a type representing a member of a group.